keyvalue-provider/src/lib.rs

```diff
@@ -1,36 +1,30 @@
 #[macro_use]
 extern crate wascc_codec as codec;
-
 #[macro_use]
 extern crate log;
 
 mod kv;
-
 use crate::kv::KeyValueStore;
-use codec::capabilities::{
-    CapabilityDescriptor, CapabilityProvider, Dispatcher, NullDispatcher, OperationDirection,
-    OP_GET_CAPABILITY_DESCRIPTOR,
-};
+use actor_core::CapabilityConfiguration;
+use actor_keyvalue::*;
+use codec::capabilities::{CapabilityProvider, Dispatcher, NullDispatcher};
 use codec::core::{OP_BIND_ACTOR, OP_REMOVE_ACTOR};
-use codec::keyvalue;
-use codec::keyvalue::*;
-use wascc_codec::core::CapabilityConfiguration;
-use wascc_codec::{deserialize, serialize};
-
 use std::error::Error;
-use std::sync::RwLock;
+use std::sync::{Arc, RwLock};
+use wascc_codec::{deserialize, serialize};
 
 #[cfg(not(feature = "static_plugin"))]
 capability_provider!(KeyvalueProvider, KeyvalueProvider::new);
 
-const CAPABILITY_ID: &str = "wascc:keyvalue";
+const CAPABILITY_ID: &str = "wasmcloud:keyvalue";
 const SYSTEM_ACTOR: &str = "system";
 const VERSION: &str = env!("CARGO_PKG_VERSION");
-const REVISION: u32 = 0; // Increment for each crates publish
+const REVISION: u32 = 1; // Increment for each crates publish
```

Comment:
these constants shouldn't be needed anymore because we no longer support the descriptor concept. Clippy should probably have flagged this as unused, right?